Model/RecoveryItem.cs

```diff
@@ -9,6 +9,7 @@ namespace CustomWatchdog
     internal class RecoveryItem
     {
         public string RecoveryBatch { get; set; }
+        public uint overrideRecoveryExecutionTimeout { get; set; }
```

Comment:
I know that this repo might need some cleaning up, but lets try to keep to our coding standards, https://github.com/Starcounter/CompanyTrack/blob/15235f13eee21b97625fe6845063d92d68d63516/AppsTeam/Guidelines/coding_style/csharp_code_formatting.md

This should be called `OverrideRecoveryExecutionTimeout`

Reply:
Done